11-bug_tracking.Rmd

```diff
@@ -1,23 +1,125 @@
+
+2. The bug report for $\textsf{R}$ Core-supported packages and/or their documentations can be submitted either to $\textsf{R}$'s [Bugzilla](https://bugs.r-project.org/bugzilla/) or as an issue to the [GitHub Mirror of R svn server](https://github.com/r-devel/r-svn/issues).
+
+3. If you want to submit the bug report using Bugzilla, please ensure that you have a Bugzilla account. To get a Bugzilla account, please send an e-mail to `bug-report-request@r-project.org` from the address that you want to use as your login. In this e-mail, briefly explain why you need an account. A volunteer shall then create a Bugzilla account and add you to $\textsf{R}$'s Bugzilla members.
+
+4. Please ensure whether the bug is already fixed (in the upcoming changes in R) or reported (search for it from those already reported on Bugzilla, either on [search existing bug reports](https://bugs.r-project.org/bugzilla/query.cgi), may use the [advanced search](https://bugs.r-project.org/bugzilla/query.cgi?format=advanced) option here,  or [show open bugs new-to-old](https://bugs.r-project.org/bugzilla/buglist.cgi?bug_file_loc_type=allwordssubstr&bug_status=NEW&bug_status=ASSIGNED&bug_status=CONFIRMED&bug_status=REOPENED&bug_status=UNCONFIRMED&bugidtype=include&chfieldto=Now&cmdtype=doit&emailassigned_to1=1&emailassigned_to2=1&emailcc2=1&emailreporter2=1&emailtype1=substring&emailtype2=substring&field0-0-0=noop&long_desc_type=substring&order=bugs.delta_ts%20desc&query_format=advanced&short_desc_type=allwordssubstr&type0-0-0=noop)). If you are submitting the report as an issue on the GitHub Mirror, then please check the currently $\textit{open}$ and $\textit{closed}$ issues first, to find if it is already reported or resolved. Do not submit a report in such a case. 
+
+### Bug in the $\textsf{R}$ Language {#RLangBug}
+
+1. If there is a bug in the $\textsf{R}$ language itself, then you need to report using the same procedure and precautions as explained [above](#RCorePkgBug).
+
+### Bug in the non $\textsf{R}$ Core-supported packages and/or their documentations {#nonRCorePkgBug}
+
+1. For packages that are not maintained by the $\textsf{R}$ Core, the bug reports are usually submitted to their maintainer(s). You can find who is the package maintainer by running either of the commands `maintainer("package_name")` or `packageDescription("package_name")`.
+
+2. In $\textsf{R}$ the running function `bug.report(package = "package_name")` shall direct you to either the GitHub issue tracker of the package, or to the bug tracking web page, or towards composing an e-mail to the package maintainer. This function `bug.report` is disabled in $\textbf{RStudio}$.
+
+3. Please ensure that your bug is not already reported or fixed before reporting it in any of the ways suggested above.

```

Comment:
If this paragraph is for #nonRCorePkgBug, #RCorePkgBug and the #RLangBug section it should be on a different section or out of this. Perhaps it could be expanded to explain how to ensure that the bug is not already reported or fixed and what to do in those occasions that it is already solved/reported. Some suggestions, search error or function name not the whole code.